tests/format/js/comments/export.js

```diff
@@ -20,3 +20,21 @@ export {
   fooo, // comment
   barr, // comment
 }
+
+const foooo = ''
+const barrr = ''
+export {
+  foooo,
+
+  barrr as  // comment
+		 baz,
+} from 'foo'
+
+const fooooo = ''
+const barrrr = ''
+export {
+  fooooo,
+
+  barrrr as  // comment
+		 bazz,
+}
```

Comment:
Can we add `import` test here too? Or maybe in other better place? So we can notice the change between import/export in one place.

Reply:
Thank !

> Can we add import test here too? Or maybe in other better place?

I created `export-and-import.js` and added tests to compare comment formats in `export` and `import` in it.
Because I thought `export.js` should only content about `exports`.